convert2rhel/unit_tests/checks_test.py

```diff
@@ -206,7 +188,7 @@ def test_convert2rhel_latest_out_of_date_el6(caplog, convert2rhel_latest_version
     local_version, dummy_ = convert2rhel_latest_version_test
     log_msg = (
         "You are currently running %s and the latest version of convert2rhel is 0.22.\n"
-        "Only the latest version is supported for conversion." % (local_version)
+        "We encurage you to update to the latest version" % local_version
```

Comment:
```suggestion
        "We encurage you to update to the latest version." % local_version
```

Reply:
Thanks for spotting all of those grammar errors. I must have been off my game or didn't go back and spell/grammar check like I usually do!